2023-07-21 13:30:13.934 [info] <0.44.0> Application lager started on node nonode@nohost
2023-07-21 13:30:13.938 [info] <0.44.0> Application xmerl started on node nonode@nohost
2023-07-21 13:30:13.945 [info] <0.44.0> Application sasl started on node nonode@nohost
2023-07-21 13:30:13.945 [info] <0.44.0> Application tools started on node nonode@nohost
2023-07-21 13:30:13.945 [info] <0.44.0> Application jsx started on node nonode@nohost
2023-07-21 13:30:13.945 [info] <0.44.0> Application recon started on node nonode@nohost
2023-07-21 13:30:14.010 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2023-07-21 13:30:14.010 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2023-07-21 13:30:14.071 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2023-07-21 13:30:14.091 [info] <0.44.0> Application amqp started on node nonode@nohost
2023-07-21 13:30:14.091 [info] <0.44.0> Application ex_rabbit started on node nonode@nohost
2023-07-21 14:32:47.531 [error] <0.276.0> gen_server <0.276.0> terminated with reason: socket_closed_unexpectedly
2023-07-21 14:32:47.531 [error] <0.276.0> CRASH REPORT Process <0.276.0> with 0 neighbours exited with reason: socket_closed_unexpectedly in gen_server:handle_common_reply/8 line 1241
2023-07-21 14:32:47.531 [error] <0.274.0> Supervisor {<0.274.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.275.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.276.0> exit with reason socket_closed_unexpectedly in context child_terminated
2023-07-21 14:32:47.532 [error] <0.274.0> Supervisor {<0.274.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.275.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.276.0> exit with reason reached_max_restart_intensity in context shutdown
